Mealie 3.15 update fails

### ✅ Have you read and understood the above guidelines?

yes

### 🔎 Did you run the script with verbose mode enabled?

Yes, verbose mode was enabled and the output is included below

### 📜 What is the name of the script you are using?

Mealie

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/mealie.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 13

### 📈 Which Proxmox version are you on?

pve-manager/9.1.6/71482d1833ded40a (running kernel: 6.17.13-2-pve)

### 📝 Provide a clear and concise description of the issue.

Updating to Malie 3.15 (from 3.4) fails and quits

### 🔄 Steps to reproduce the issue.

Execute the update script from within the LXC.

### ❌ Paste the full error output (if available).

```
root@mealie:~# cat /tmp/create-lxc-ad35cb11.log
[2026-04-13 15:45:55] [INFO] Checking for update: mealie
[2026-04-13 15:45:56] [OK] Update available: mealie 3.4.0 → 3.15.0
[2026-04-13 15:45:56] [INFO] Upgrade uv from 0.9.7 to 0.11.6
[2026-04-13 15:45:58] [INFO] Installing Python 3.12 via uv
[2026-04-13 15:45:59] [OK] Python 3.12 installed
[2026-04-13 15:45:59] [OK] Setup uv 0.11.6
[2026-04-13 15:45:59] [INFO] Upgrade Node.js from 20 to 24
[2026-04-13 15:46:23] [OK] Setup Node.js 24
[2026-04-13 15:46:24] [INFO] Updating yarn to latest version
[2026-04-13 15:46:25] [OK] Installed Node.js modules: yarn
[2026-04-13 15:46:25] [INFO] Stopping Service
[2026-04-13 15:46:25] [OK] Stopped Service
[2026-04-13 15:46:25] [INFO] Backing up Configuration
[2026-04-13 15:46:25] [OK] Backup completed
[2026-04-13 15:46:26] [INFO] Fetching GitHub release: mealie (3.15.0)
[2026-04-13 15:46:35] [OK] Deployed: mealie (3.15.0)
[2026-04-13 15:46:35] [INFO] Installing Python Dependencies with uv
[2026-04-13 15:46:44] [OK] Installed Python Dependencies
[2026-04-13 15:46:44] [INFO] Building Frontend
[2026-04-13 15:46:44] [ERROR] in line 52: exit code 2 (Misuse of shell builtins (e.g. syntax error)): while executing command  sed -i "s|https://github.com/mealie-recipes/mealie/commit/|https://github.com/mealie-recipes/mealie/releases/tag/|g" /opt/mealie/frontend/pages/admin/site-settings.vue
[2026-04-13 15:46:44] Full log: /tmp/create-lxc-ad35cb11.log
```

### 🖼️ Additional context (optional).

Fails because /opt/mealie/frontend/pages/admin/site-settings.vue isn't there everything from pages down is missing

```root@mealie:/opt/mealie/frontend# ls -al
total 377
drwxr-xr-x  5 root root     17 Apr 13 15:55 .
drwxr-xr-x 13 root root     29 Apr 13 15:55 ..
drwxr-xr-x 13 root root     14 Apr 13 15:55 app
-rw-r--r--  1 root root    207 Apr 13 15:55 .editorconfig
-rw-r--r--  1 root root    754 Apr 13 15:55 eslint.config.mjs
-rw-r--r--  1 root root   1253 Apr 13 15:55 .gitignore
-rw-r--r--  1 root root  12761 Apr 13 15:55 nuxt.config.ts
-rw-r--r--  1 root root     14 Apr 13 15:55 .nuxtignore
-rw-r--r--  1 root root   1794 Apr 13 15:55 package.json
-rw-r--r--  1 root root    163 Apr 13 15:55 .prettierrc
drwxr-xr-x  5 root root     16 Apr 13 15:55 public
drwxr-xr-x  4 root root      4 Apr 13 15:55 server
-rw-r--r--  1 root root    553 Apr 13 15:55 template.env
-rw-r--r--  1 root root    131 Apr 13 15:55 tsconfig.json
-rw-r--r--  1 root root    376 Apr 13 15:55 vitest.config.js
-rw-r--r--  1 root root   2013 Apr 13 15:55 vuetify.options.js
-rw-r--r--  1 root root 520772 Apr 13 15:55 yarn.lock
```

Possibly has to do with the change from nuxt 3 to 4 that 3.15 bring? https://github.com/mealie-recipes/mealie/releases/tag/v3.15.0

--- a/ct/mealie.sh
+++ b/ct/mealie.sh
@@ -28,8 +28,7 @@ function update_script() {
     msg_error "No ${APP} Installation Found!"
     exit
   fi
-  RELEASE="v3.14.0"
-  if check_for_gh_release "mealie" "mealie-recipes/mealie" "${RELEASE}" "each release is tested individually before the version is updated. Please do not open issues for this"; then
+  if check_for_gh_release "mealie" "mealie-recipes/mealie"; then
     PYTHON_VERSION="3.12" setup_uv
     NODE_MODULE="yarn" NODE_VERSION="24" setup_nodejs
 
@@ -41,7 +40,7 @@ function update_script() {
     cp -f /opt/mealie/mealie.env /opt/mealie.env
     msg_ok "Backup completed"
 
-    CLEAN_INSTALL=1 fetch_and_deploy_gh_release "mealie" "mealie-recipes/mealie" "tarball" "${RELEASE}" "/opt/mealie"
+    CLEAN_INSTALL=1 fetch_and_deploy_gh_release "mealie" "mealie-recipes/mealie" "tarball" "latest" "/opt/mealie"
 
     msg_info "Installing Python Dependencies with uv"
     cd /opt/mealie
@@ -50,9 +49,10 @@ function update_script() {
 
     msg_info "Building Frontend"
     MEALIE_VERSION=$(<$HOME/.mealie)
-    $STD sed -i "s|https://github.com/mealie-recipes/mealie/commit/|https://github.com/mealie-recipes/mealie/releases/tag/|g" /opt/mealie/frontend/pages/admin/site-settings.vue
-    $STD sed -i "s|value: data.buildId,|value: \"v${MEALIE_VERSION}\",|g" /opt/mealie/frontend/pages/admin/site-settings.vue
-    $STD sed -i "s|value: data.production ? i18n.t(\"about.production\") : i18n.t(\"about.development\"),|value: \"bare-metal\",|g" /opt/mealie/frontend/pages/admin/site-settings.vue
+    SITE_SETTINGS=$(find /opt/mealie/frontend -name "site-settings.vue" -path "*/admin/*" | head -1)
+    $STD sed -i "s|https://github.com/mealie-recipes/mealie/commit/|https://github.com/mealie-recipes/mealie/releases/tag/|g" "$SITE_SETTINGS"
+    $STD sed -i "s|value: data.buildId,|value: \"v${MEALIE_VERSION}\",|g" "$SITE_SETTINGS"
+    $STD sed -i "s|value: data.production ? i18n.t(\"about.production\") : i18n.t(\"about.development\"),|value: \"bare-metal\",|g" "$SITE_SETTINGS"
     export NUXT_TELEMETRY_DISABLED=1
     cd /opt/mealie/frontend
     $STD yarn install --prefer-offline --frozen-lockfile --non-interactive --production=false --network-timeout 1000000
@@ -98,4 +98,3 @@ msg_ok "Completed successfully!\n"
 echo -e "${CREATING}${GN}${APP} setup has been successfully initialized!${CL}"
 echo -e "${INFO}${YW} Access it using the following URL:${CL}"
 echo -e "${TAB}${GATEWAY}${BGN}http://${IP}:9000${CL}"
-

--- a/ct/mealie.sh
+++ b/ct/mealie.sh
@@ -40,7 +40,7 @@ function update_script() {
     cp -f /opt/mealie/mealie.env /opt/mealie.env
     msg_ok "Backup completed"
 
-    CLEAN_INSTALL=1 fetch_and_deploy_gh_release "mealie" "mealie-recipes/mealie" "tarball" "latest" "/opt/mealie"
+    CLEAN_INSTALL=1 fetch_and_deploy_gh_release "mealie" "mealie-recipes/mealie" "tarball"
 
     msg_info "Installing Python Dependencies with uv"
     cd /opt/mealie

--- a/install/mealie-install.sh
+++ b/install/mealie-install.sh
@@ -30,7 +30,7 @@ msg_ok "Installed Dependencies"
 PYTHON_VERSION="3.12" setup_uv
 PG_VERSION="16" setup_postgresql
 NODE_MODULE="yarn" NODE_VERSION="24" setup_nodejs
-fetch_and_deploy_gh_release "mealie" "mealie-recipes/mealie" "tarball" "v3.14.0" "/opt/mealie"
+fetch_and_deploy_gh_release "mealie" "mealie-recipes/mealie" "tarball" "latest" "/opt/mealie"
 PG_DB_NAME="mealie_db" PG_DB_USER="mealie_user" PG_DB_GRANT_SUPERUSER="true" setup_postgresql_db
 
 msg_info "Installing Python Dependencies with uv"
@@ -42,9 +42,10 @@ msg_info "Building Frontend"
 MEALIE_VERSION=$(<$HOME/.mealie)
 export NUXT_TELEMETRY_DISABLED=1
 cd /opt/mealie/frontend
-$STD sed -i "s|https://github.com/mealie-recipes/mealie/commit/|https://github.com/mealie-recipes/mealie/releases/tag/|g" /opt/mealie/frontend/pages/admin/site-settings.vue
-$STD sed -i "s|value: data.buildId,|value: \"v${MEALIE_VERSION}\",|g" /opt/mealie/frontend/pages/admin/site-settings.vue
-$STD sed -i "s|value: data.production ? i18n.t(\"about.production\") : i18n.t(\"about.development\"),|value: \"bare-metal\",|g" /opt/mealie/frontend/pages/admin/site-settings.vue
+SITE_SETTINGS=$(find /opt/mealie/frontend -name "site-settings.vue" -path "*/admin/*" | head -1)
+$STD sed -i "s|https://github.com/mealie-recipes/mealie/commit/|https://github.com/mealie-recipes/mealie/releases/tag/|g" "$SITE_SETTINGS"
+$STD sed -i "s|value: data.buildId,|value: \"v${MEALIE_VERSION}\",|g" "$SITE_SETTINGS"
+$STD sed -i "s|value: data.production ? i18n.t(\"about.production\") : i18n.t(\"about.development\"),|value: \"bare-metal\",|g" "$SITE_SETTINGS"
 $STD yarn install --prefer-offline --frozen-lockfile --non-interactive --production=false --network-timeout 1000000
 $STD yarn generate
 msg_ok "Built Frontend"

--- a/install/mealie-install.sh
+++ b/install/mealie-install.sh
@@ -30,7 +30,7 @@ msg_ok "Installed Dependencies"
 PYTHON_VERSION="3.12" setup_uv
 PG_VERSION="16" setup_postgresql
 NODE_MODULE="yarn" NODE_VERSION="24" setup_nodejs
-fetch_and_deploy_gh_release "mealie" "mealie-recipes/mealie" "tarball" "latest" "/opt/mealie"
+fetch_and_deploy_gh_release "mealie" "mealie-recipes/mealie" "tarball"
 PG_DB_NAME="mealie_db" PG_DB_USER="mealie_user" PG_DB_GRANT_SUPERUSER="true" setup_postgresql_db
 
 msg_info "Installing Python Dependencies with uv"

